dcos_installer/action_lib.py

```diff
@@ -373,18 +373,48 @@ def uninstall_dcos(config, block=False, state_json_dir=None, async_delegate=None
 def _add_prereqs_script(chain):
     inline_script = """
 #/bin/sh
-# setenforce is in this path
-PATH=$PATH:/sbin
 
-dist=$(cat /etc/os-release | sed -n 's@^ID="\(.*\)"$@\\1@p')
+# For setenforce & xfs_info
```

Comment:
Is `xfs_info` being used in the old code? If not, `/sbin` should be enough...

Reply:
`wfs_info` is NOT used by the old code. It was added by my changes to master: https://github.com/dcos/dcos/blob/master/dcos_installer/action_lib.py#L543